pymc/tests/test_mixture.py

```diff
@@ -657,6 +658,28 @@ def test_iterable_single_component_warning(self):
         with pytest.warns(UserWarning, match="Single component will be treated as a mixture"):
             Mixture.dist(w=[0.5, 0.5], comp_dists=[Normal.dist(size=2)])
 
+    @pytest.mark.parametrize(
+        "weights, comp_dists, size, expected",
+        [
+            (
+                np.array([1, 0]),
+                [Normal.dist(-2, 5), Normal.dist(2, 5)],
```

Comment:
We'll need a couple more variations with multidimensional distributions x single components x sizes

Reply:
Yes! This was a temporary template since I ran into a bug (due to a typo...)